walkthroughs/howto-k8s-outlier-detection/README.md

```diff
@@ -0,0 +1,361 @@
+
+Let's generate traffic to verify the behavior
+
+```
+echo "GET http://front.howto-k8s-outlier-detection:8080/color/get" | vegeta attack -duration=4s | tee results.bin | vegeta report
+
+Requests      [total, rate, throughput]         200, 50.25, 48.97
+Duration      [total, attack, wait]             3.982s, 3.98s, 2.281ms
+Latencies     [min, mean, 50, 90, 95, 99, max]  1.649ms, 2.178ms, 2.062ms, 2.378ms, 2.467ms, 8.118ms, 12.213ms
+Bytes In      [total, mean]                     1435, 7.17
+Bytes Out     [total, mean]                     0, 0.00
+Success       [ratio]                           97.50%
+Status Codes  [code:count]                      200:195  500:5
+Error Set:
+500 Internal Server Error
+```
+
+Notice that there are 5 requests that returned HTTP 500 response in the `Status Codes` row. App Mesh should detect this and eject the host for 10s. For next 10s, you should see all requests return HTTP 200 responses as the faulty host will no longer be serving traffic.
+
+Let's look at the stats to verify this:
+
+```
+curl front.howto-k8s-outlier-detection:8080/stats | jq .
+
+[
+  {
+    "HostUID": "8f04b1c8-af29-4345-8a0d-34cb5c981e38",
+    "Counter": {
+      "StatusOk": 118,
+      "StatusError": 0,
+      "Total": 118
+    }
+  },
+  {
+    "HostUID": "34bb223d-1e6c-4423-898e-372d30a638b2",
+    "Counter": {
+      "StatusOk": 118,
+      "StatusError": 0,
+      "Total": 118
+    }
+  },
+  {
+    "HostUID": "c87a6e70-c9a2-4343-a453-81808bec9d2d",
+    "Counter": {
+      "StatusOk": 52,
+      "StatusError": 5,
+      "Total": 57
+    }
+  },
+  {
+    "HostUID": "c3338a28-8590-48e6-9c53-77c4e15100dc",
+    "Counter": {
+      "StatusOk": 117,
+      "StatusError": 0,
+      "Total": 117
+    }
+  }
+]
+```
+
+Notice that host `c87a6e70-c9a2-4343-a453-81808bec9d2d` returned 5 errors and no traffic was sent to this host. All the traffic was distributed between the other three hosts.
+
+If we send more traffic during the ejection duration, we will see that all the requests will return `HTTP 200` responses since only the healthy hosts will be serving traffic:
+
+```
+echo "GET http://front.howto-k8s-outlier-detection:8080/color/get" | vegeta attack -duration=4s | tee results.bin | vegeta report
+
+Requests      [total, rate, throughput]         200, 50.25, 50.23
+Duration      [total, attack, wait]             3.982s, 3.98s, 1.945ms
+Latencies     [min, mean, 50, 90, 95, 99, max]  1.605ms, 1.995ms, 1.962ms, 2.267ms, 2.318ms, 2.441ms, 5.114ms
+Bytes In      [total, mean]                     1400, 7.00
+Bytes Out     [total, mean]                     0, 0.00
+Success       [ratio]                           100.00%
+Status Codes  [code:count]                      200:200
+Error Set:
+```
+
+The request above was made while the host `c87a6e70-c9a2-4343-a453-81808bec9d2d` was ejected and we got 100% successful responses.
+
```

Comment:
Might be good to show the bad host/instance of the color app will be added back after the ejection duration.

Reply:
++, added!